src/Form/Form.js

```diff
@@ -88,14 +88,18 @@ class Form extends Util.mixin(BindMixin) {
       nextState.model = mergeNewModel(state.model, currentModel, nextModel);
     }
 
-    let currentErrors = this.buildStateObj(props.definition, 'showError');
-    let nextErrors = this.buildStateObj(nextProps.definition, 'showError');
-    if (!Util.isEqual(currentErrors, nextErrors)) {
-      nextState.erroredFields = mergeNewModel(
-        state.erroredFields,
-        currentErrors,
-        nextErrors
-      );
+    if (this.props.useExternalErrors) {
```

Comment:
Do we really need this prop? Shouldn't it be enough to always render the errors from the definition if defined?